WBooks/Helpers/LineTextField.swift

```diff
@@ -24,12 +26,6 @@ class LineTextField: UITextField {
         _ = add(bottom: .init(thickness: 0.5, color: GeneralConstants.Design.textfieldGrayColor))
     }
     
-    required override init(frame: CGRect) {
```

Comment:
Just checking: This said required, are you sure you don't need it?

Reply:
Yes, I already have a `required init`